floss/language/go/extract.py

```diff
@@ -308,7 +630,7 @@ def extract_go_strings(sample: Path, min_length=MIN_STR_LEN) -> List[StaticStrin
     return utf_8_parts + stack_strings + static_strings
 
 
-def main(argv=None):
+def amain(argv=None):
```

Comment:
temporarily disabled the existing main while i was testing. will re-enable later.

Reply:
need to change this again